cpu/native/include/netdev_raw802154.h

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (C) 2015 Kaspar Schleiser <kaspar@schleiser.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for
+ * more details.
+ */
+
+/**
+ * @ingroup     netdev
+ * @brief       Low-level ethernet driver for native tap interfaces
+ * @{
+ *
+ * @file
+ * @brief       Definitions for @ref netdev ethernet driver for host system's
+ *              TAP interfaces
+ *
+ * @author      Kaspar Schleiser <kaspar@schleiser.de>
+ */
+#ifndef NETDEV_RAW802154_H
+#define NETDEV_RAW802154_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include "net/netdev.h"
+#include "net/netdev/ieee802154.h"
+
+#ifdef __MACH__
+#include "net/if_var.h"
+#else
+#include "net/if.h"
+#endif
+
+/**
+ * @brief tap interface state
+ */
+typedef struct netdev_raw802154 {
+    netdev_ieee802154_t netdev;
```

Comment:
I think the doc generator will complain, if these members aren't documented (you can use `/**< comment */` comments for that. But I'm not sure